### PR TITLE
doc: use 3 issue template for better community experience

### PR DESCRIPTION
**Pull request by imbajin**
At present, there are 3 issue template and 1 config file:
- Bug Report
- Feature Request
- Ask question (Guide)

And simplify the issue template to improve the user's willingness to fill in, like this:

image

BUG like:
image

Question like:
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 是否允许创建空白的 issue
+blank_issues_enabled: true
+
+# 设置提 issue 前的参考文档
+contact_links:
+  - name: HugeGraph Server Doc
+    url: https://hugegraph.github.io/hugegraph-doc/quickstart/hugegraph-server.html
+    about: Please search question here before open a issue
+  - name: HugeGraph API Doc
+    url: https://hugegraph.github.io/hugegraph-doc/quickstart/hugegraph-server.html
```
Comment (zhoney) [inline .github/ISSUE_TEMPLATE/config.yml:10]: change to API link

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Actual behavior ( 实际表现 / 报错)
+
+xxx
+
+```java
+// Error info ⬇ (尽可能详细的日志 + 完整异常栈)
+
+```
+
+## How to reproduce ( 复现方式 )
```
Comment (zhoney) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: 复现方式->复现步骤

**Review by zhoney — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+labels: 'bug'
+assignees: ''
+
+---
+
+### Note ( 特别注意 ) : 
+
+> 1. 请先**搜索**, 并**确认**现有的 [issue](https://github.com/hugegraph/hugegraph/issues) 与 [FAQ](https://hugegraph.github.io/hugegraph-doc/guides/faq.html) 中没有与您相同 / 相关的问题, 勿重复提交
+> 2. 我们需要尽可能**详细**的信息来**复现**问题, 越详细的信息 (包括**日志 / 截图 / 配置**等) 会**越快**被响应和处理
+> 3. 请关注提交的 issue, 缺乏信息 / 长时间 ( > 14 天) 没有回复, issue 可能会被 **关闭**
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: 增加：被关闭之后，可以随时 reopen 重新打开。

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+name: Bug report (反馈 Bug)
+about: Create a bug report to help HugeGraph improve
+title: '[BUG] short description'
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: improve "short description", if it to be filled, like "Type summary description here" is ok

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+labels: 'bug'
+assignees: ''
+
+---
+
+### Note ( 特别注意 ) : 
+
+> 1. 请先**搜索**, 并**确认**现有的 [issue](https://github.com/hugegraph/hugegraph/issues) 与 [FAQ](https://hugegraph.github.io/hugegraph-doc/guides/faq.html) 中没有与您相同 / 相关的问题, 勿重复提交
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: [Issues]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+1. step 1
+2. step 2
+3. step 3
+
+**Refer:** [How to create a minimal reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example)
+
+### Vertex/Edge example ( 问题点 / 边数据举例 )
+
+```javascript
+// JSON Vertex / Edge ⬇
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]: JSON of Vertex or Edge

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+```
+
+### Schema [VertexLabel, EdgeLabel, IndexLabel] ( 元数据结构 )
+
+```javascript
+// JSON Schema ⬇
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.md:52]: JSON of Schema

File: .github/ISSUE_TEMPLATE/config.yml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# 是否允许创建空白的 issue
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/config.yml:1]: remove this comment

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request (新需求 / 功能)
+about: Give an idea for HugeGraph
+title: '[Feature] short description'
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: ditto

File: .github/ISSUE_TEMPLATE/question_ask.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
+---
+name: Ask question (提问)
+about: Question about usage or configs in HugeGraph
+title: '[Question] short description'
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/question_ask.md:4]: ditto

**Review by javeme — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**